crdppf/static/js/Crdppf/featurePanel.js

```diff
@@ -0,0 +1,284 @@
+Ext.namespace('Crdppf');
+OpenLayers.ImgPath = Crdppf.OLImgPath;  
+
+// Constructor
+Crdppf.FeaturePanel = function Map() {    
+    this.init();
+};
+
+Crdppf.FeaturePanel.prototype = {
+    /**
+    * Property: currently selected property
+    **/
+    currentProperty: null,
+    /**
+    * Object: Feature tree to display intersection results
+    * Parameters:
+    * none
+    */
+    featureTree: new Ext.tree.TreePanel ({
+        title: Crdppf.labels.restrictionPanelTitle,
+        cls: 'featureTreeCls',
+        collapsed: false,
+        useArrows:false,
+        collapside: false,
+        animate:true,
+        lines: false,
+        enableDD:false,
+        rootVisible: false,
+        frame: false,
+        id: 'featureTree',
+        height:300,
+        autoScroll: true
+    }),
+    /**
+    * Object: Root of the featureTree
+    * Parameters:
+    * none
+    */
+    root: new Ext.tree.TreeNode ({
+        text: 'Thèmes',
+        draggable:false,
+        id:'rootNode'
+    }),
+    /**
+    * Method: disable the existing info controls
+    * Parameters:
+    * none
+    */
+    disableInfoControl: function disableInfoControl(){
+        this.featureTree.collapse(false);
+        this.root.removeAll(true);
+        this.featureTree.setTitle(Crdppf.labels.restrictionPanelTitle);
+        Crdppf.Map.selectLayer.removeAllFeatures();
+        Crdppf.Map.intersectLayer.removeAllFeatures();
+        Crdppf.Map.infoControl.deactivate();
+    },
+    /**
+    * Method: create the info control
+    * Parameters:
+    * none
+    */
+    setInfoControl: function setInfoControl(){
+        Crdppf.Map.infoControl.activate();
+        this.root.removeAll(true);
+    },
+    /***
+    * Method: On multiple property selection (DP case) the user is ask to select only one
+    * Parameters:
+    * features:
+    * labels: 
+    */
+    PropertySelection: function(features, labels) {
+        var propertySelectionWindow;
+
+        if (store) {
+            store.removeAll();
+        }
+
+        if (features.length > 1) {
+            var properties = [];
+            for (var i = 0; i < features.length; i++){
+                properties[i] = [
+                    i,
+                    features[i].data.typimm+': '+features[i].data.nummai+' '+features[i].data.cadastre,
+                    features[i].data.idemai,
+                    features[i].data.source
+                ];
+            }
+
+            var store = new Ext.data.ArrayStore({
```

Comment:
Personally I'm not a big fan of general named variables and objects, I'd prefer unique and verbose variable names to be sure not to use twice the same name and avoid potential conflicts when the code gets longer. Here I'd suggest something like 'propertiesstore'. But I leave it to you to rename the store or not :)